test/address_sanitizer_setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+OLD_DIR=`pwd`
+
+cd "$SCRIPT_DIR/.."
+
+if [ ! -d otp ]
+then
+    git clone git@github.com:erlang/otp.git
+
+    cd otp
+
+    export ERL_TOP=`pwd`
+    export PATH=$ERL_TOP/bin:$PATH
+    
+    echo "Changed ERL_TOP and PATH"
+
+
+    git checkout OTP-24.2.1
+
+    LAST_COMMIT_SHA=$(git rev-parse HEAD)
+
+    if [ ${LAST_COMMIT_SHA} != 7bf7f01683acf9b8f09bd8c7331854a9abc17f7d ]
+    then
+        echo "Unexpected commit SHA"
+        exit 1
+    fi
+
+    ./configure
+
+    make -j
+
+    cd erts/emulator
+
+    make asan
+fi
+
+cd "$SCRIPT_DIR/../otp"
+
+export ERL_TOP=`pwd`
+export PATH=$ERL_TOP/bin:$PATH
+
+echo Changed ERL_TOP and PATH
+
+cd "$SCRIPT_DIR/.."
+
+rm -rf c_src/jqc
+rm -rf c_src/ext_libs
+
+rebar3 clean
+
+(mkdir c_src/ext_libs || true)
+
+cp "$ERL_TOP/lib/erl_interface/obj/x86_64-pc-linux-gnu/libei.a" c_src/ext_libs/
+
+JQ_MEMSAN_DEBUG=1 rebar3 compile
+
+cd "$OLD_DIR"
```

Comment:
Nit: probably we don't need to remember the old dir, unless this script is to be sourced.

Reply:
Thanks will fix!

Reply:
Fixed in commit below.